Source/Utilities/Slice.lua

```diff
@@ -0,0 +1,9 @@
+function Auctionator.Utilities.Slice(array, start, count)
```

Comment:
We discussed this a while back - this is supposedly nasty in terms of memory usage. We should use a LL if we really want to segment lists efficiently... We can come back to this later but just want to make sure we flag this (once we finish implementing everything, our next order of business will be to make this thing more efficient, especially since we have a huge chunk of data in memory).

Reply:
It starts as an array, so we'd need to convert it to a linked list, and back again just to make it work with the API (the WoW search API doesn't do linked lists...)

Reply:
It is "only" 100 items long, this could be expensive or mess with a GC if we did this every frame for really long lists, but its pretty short and (anecdotally) the searches are done every few minutes, nowhere near the frame limit